precompiles/testutil/logs.go

```diff
@@ -40,7 +40,7 @@ func CheckLogs(logArgs LogCheckArgs) error {
 		)
 	}
 
-	if err := CheckVMError(logArgs.Res, logArgs.ErrContains); err != nil {
+	if err := CheckVMError(logArgs.Res, logArgs.ErrContains); err != nil { //nolint:govet
```

Comment:
add an explainer to why things are escaped please
```suggestion
	if err = CheckVMError(logArgs.Res, logArgs.ErrContains); err != nil { //nolint:govet // fmt string is okay to be passed to this function
```